### PR TITLE
Add vendored config-rules

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -121,3 +121,13 @@ terratest/install: | guard/program/go guard/program/dep
...
 terratest/test: | guard/program/go guard/program/dep
 	cd tests && go test -timeout 20m
+
+clean:
+	rm -rf vendor
+
+VENDOR ?= vendor/github.com/awslabs/aws-config-rules
+vendor: $(VENDOR)
+
+vendor/%:
```
Comment (userhas404d) [inline Makefile]: I guess the answer is 'just don't forget to run it manually before a release' but would it make sense to include this as part of the release workflow?

Comment (lorengordon) [reply Makefile]: not so much _every_ release, as just periodically, when there are new or updated config rules we want to use... then it's just: `make clean && make vendor`. worth putting in the README though

**Comment by lorengordon** [comment]
Oooh, yeah, can exclude the vendor directory from linting... Don't want to modify those at all.

File: Makefile (hunk 1 above)
Comment (confusdcodr) [reply Makefile]: Added to the README

File: vendor/.editorconfig [hunk 2]
```diff
@@ -0,0 +1 @@
+root = true
```
Comment (lorengordon) [inline vendor/.editorconfig]: `make clean` will delete this... can update the `vendor` target to create it...

```
vendor: $(VENDOR)
	echo "root = true" > vendor/.editorconfig
```

Comment (confusdcodr) [reply vendor/.editorconfig]: Fixed

**Pull request merged**